src/main/resources/bamm/meta-model/2.0.0/aspect-meta-model-shapes.ttl

```diff
@@ -201,32 +216,43 @@ bamm:EntityShape
                 group by $this
               }
               filter ( !bound( ?propertyCount ) || ( ?propertyCount = 0 ) )
+              bind( 'ERR_INVALID_PROPERTY' as ?code )
             }
          """
       ] ;
       sh:sparql [
          a sh:SPARQLConstraint ;
-         sh:message "A Property of an Entity may not be defined with both bamm:optional \"true\"^^xsd:boolean and bamm:notInPayload \"true\"^^xsd:boolean." ;
+         sh:message "Property '{?value}' of the Entity '{$this}' may not be defined with both bamm:optional \"true\"^^xsd:boolean and bamm:notInPayload \"true\"^^xsd:boolean." ;
```

Comment:
"Property '{?value}' of the Entity '{$this}' must not be defined as both bamm:optional \"true\"^^xsd:boolean or bamm:notInPayload \"true\"^^xsd:boolean."